spec/controllers/announcements_controller_spec.rb

```diff
@@ -130,4 +145,43 @@
       it_behaves_like 'access denied'
     end
   end
+  context 'check hide' do
+    before { @announcement = FactoryGirl.create(:announcement) }
+    context 'superuser' do
```

Comment:
Let's add "as" to the front of the string so it reads correctly, e.g. `GET hide as superuser can hide announcement...`